datasets/hebrew_this_world/hebrew_this_world.py

```diff
@@ -0,0 +1,95 @@
+"""HebrewThisWorld: A corpus from https://thisworld.online/."""
+
+from __future__ import absolute_import, division, print_function
+
+import csv
+import ctypes
+
+import datasets
+
+
+_DESCRIPTION = """\
+HebrewThisWorld is a data set consists of 2028 issues of the newspaper 'This World' edited by Uri Avnery and were published between 1950 and 1989. Released under the AGPLv3 license."""
+
+csv.field_size_limit(int(ctypes.c_ulong(-1).value // 2))
+
+_TRAIN_DOWNLOAD_URLS = [
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_0.csv",
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_1.csv",
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_2.csv",
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_3.csv",
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_4.csv",
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_5.csv",
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_6.csv",
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_7.csv",
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_8.csv",
+    "https://github.com/imvladikon/datasets_additional/raw/master/data/thisworld1/metadata_9.csv",
+]
+
+
+class HebrewThisWorld(datasets.GeneratorBasedBuilder):
+    """HebrewThisWorld: Corpus from the newspaper ThisWorld"""
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "issue_num": datasets.Value("string"),
+                    "page_count": datasets.Value("string"),
```

Comment:
I think these two features can be `datasets.Value("int64")`

Reply:
fixed